README.md

```diff
@@ -127,6 +127,56 @@ On Windows 10, `[options]` should be a String or an Object.
 
         If true, log will be printed to console.
 
+## Known Issues
+
+* Known issue on Windows 11 - opening web developer tools will disable the acrylic effect.
+
```

Comment:
I don't think that this specific issue needs to documented in the readme. All the issues are already documented in https://github.com/Seo-Rii/electron-acrylic-window/issues.